plugins/modules/cloud/misc/terraform.py

```diff
@@ -80,8 +80,13 @@
     aliases: [ 'variables_file' ]
   variables:
     description:
-      - A group of key-values to override template variables or those in
-        variables files.
+      - A group of key-values to override template variables or those in variables files.
+      - Support complex variable structures to reflect terraform variable syntax.
+      - Ansible dictionaries are mapped to terraform objects.
+      - Ansible lists are mapped to terraform lists.
+      - Ansible integers or floats are mapped to terraform numbers.
+      - Ansible booleans are mapped to terraform booleans.
+      - "B(Note) passwords passed as variables will be visible in the log output. Make sure to use C(no_log: true) in production!"
```

Comment:
Although it is not strictly referring to one of the module's options, I think it still fits that bill:
```suggestion
      - "B(Note) passwords passed as variables will be visible in the log output. Make sure to use I(no_log=true) in production!"
```
Per https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation

Reply:
This is definitely an edge case, I've seen both formattings for keywords, and when I quickly grepped around it seems both are equally common.

Reply:
I thought so. Thanks for checking. :)